test/e2e/page-objects/pages/home/asset-list.ts

```diff
@@ -235,8 +248,20 @@ class AssetListPage {
       this.tokenImportSelectNetwork(chainId),
     );
     await this.driver.fill(this.tokenAddressInput, tokenAddress);
-    await this.driver.fill(this.tokenSymbolInput, symbol);
+    await this.driver.waitForSelector(this.tokenSymbolTitle);
+
+    if (symbol) {
+      // do not fill the form until the button is disabled, because there's a form re-render which can clear the input field causing flakiness
+      await this.driver.waitForSelector(this.importTokensNextButton, {
+        state: 'disabled',
+        waitAtLeastGuard: 1000,
```

Comment:
we need a guard because the state of the button is:
1. disabled
2. brief ms enabled
3. disabled

it's something that could be reviewed on the app side but we can mitigate this behaviour, by using the guard